Gruntfile.js

```diff
@@ -112,7 +112,7 @@ module.exports = function(grunt) {
 		},
 
 		clean: {
-			src: ["test/**/*.js*", "**/*.js*", "!vendor/*.js", "*.tgz"]
+			src: ["test/**/*.js*", "**/*.js*", "!**/*.json", "!Gruntfile.js", "!node_modules/**/*", "!bin/common-lib.js",  "!vendor/*.js", "*.tgz"]
```

Comment:
maybe it is better to specify the `.js` and `.js.map` directly  instead of using `"**/*.js*"`